packages/@lwc/wds-core/src/browser/ssr/index.js

```diff
@@ -71,3 +71,9 @@ export async function resetMock(mockedModuleUrl) {
   url.searchParams.set('env', 'ssr');
   return await task('resetMock', url.href);
 }
+
+export async function update(mockedModuleUrl, code) {
```

Comment:
Because it may be used for interactions other than updates, I'd prefer a name like `evalCode`